tests/epyccel/test_epyccel_pow.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from numpy.random import rand, randint, uniform
+from numpy import isclose
+
+from pyccel.decorators import types
+from pyccel.epyccel import epyccel
+from conftest import *
+
+def test_pow_int_int(language):
+    @types(int, int)
+    def f_call(x, y):
+        return x ** y
+
+    f = epyccel(f_call, language=language)
+    x = randint(10)
+    y = randint(10)
+
+    assert f(x, y) == f_call(x, y)
+    # negative base
+    assert f(-x, y) == f_call(-x, y)
+
+    assert isinstance(f(x, y), type(f_call(x, y)))
+
+def test_pow_real_real(language):
+    @types('real', 'real')
+    def pow_r_r(x, y):
+        return x ** y
+
+    f = epyccel(pow_r_r, language=language)
+    x = rand()
+    y = rand()
```

Comment:
Here we are only testing with a base `0<= x < 1` and an exponent `0 <= y < 1`. Is there a reason for restricting the test to small positive numbers?

Reply:
I used a small range to avoid overflow, and I think this range will be enough and fast in testing.
as you know pow operation consummate time especially for real numbers.
I forgot to add negative values for some tests, I will add them.

Reply:
negative cases show me a lot of interesting cases, where the c and Fortran behave in a different manner.

Reply:
Can we test at least those cases where Python, C and Fortran behave the same way? For example, if `M > 1` is a real number and`N > 1` an integer (not too big to avoid overflows):
1. Real base `-M < x < M`, and positive integer exponent `y = 0, 1, 2, ..., N` (be careful to avoid the case `0**0` though);
2. Positive real base `0 < x < M` (not zero), and real exponent `-M < y < M`.

Reply:
yah, those are interesting cases I will add them.